app/views/users/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="review-list">
+  <% @user.reviews.each do |review| %>
+  <div id="book-<%= review.book.id %>-info">
+    <img src="<%= review.book.thumbnail %>" alt="Cover of <%= review.book.title %>">
+    <br><h3><%= review.book.title %></h3><br>
+    <!-- possible format change for readability -->
+  </div>
+  <div id="review-<%= review.id %>-info">
+    <h3><%= review.title %></h3>
+    <p>Written by: <%= @user.username %></p>
+    <p>Written on: <%= Date.strptime(review.updated_at.to_s)%></p>
+    <p>Rating: <%= review.rating %>/5</p>
+    <p><%= review.text %></p>
+  </div>
+  <% end %>
+  <% if @user.reviews.count == 0 %>
+  <p>This user has written no reviews yet.</p>
```

Comment:
Indent contents of this if block for readability

Reply:
Agreed, formatting changes applied